### ISSUE TITLE
MO 2 Alpha 2 - Adminstart - "Run" wont start fo4  

### ISSUE THREAD

**Issue by Senvi** [TanninOne/modorganizer#403]
Starting Mod Organizer alpha 2 as admin
Run Fallout 4
Mo is locked
Preparing vfs loading bar runs through
Mo is unlocked
no game start

Windows 10 Enterprise x64
[usvfs-2015-11-30_11-58-07.txt](https://github.com/TanninOne/modorganizer/files/47248/usvfs-2015-11-30_11-58-07.txt)
[mo_interface.txt](https://github.com/TanninOne/modorganizer/files/47249/mo_interface.txt)

no big deal works fine without admin

**Comment by fireundubh** [comment]
This is a dupe of #399 albeit for WIndows 10.

**Comment by TanninOne** [comment]
No, this is a different issue. In #399 the game does start but crashes after a few seconds. Here it doesn't look as if the vfs lib ever gets loaded in the remote process.

**Comment by HarperZ9** [comment]
I am having the same problem. Mod Organizer and all of my steam files are in an entirely different directory than C: ; Whether Vanilla or Modded the game seems to have an issue booting through MO2. Loads all the way to the the end of the bar, and seems as if something does not hook. When it attempts to launch through SKSE, the cmd stays blank until I finally unlock MO and use task manager to track down the running fallout exe and disable it. Spent all night trying to get it to work but am getting tired now, so I will work on it some more tomorrow.

**Comment by TheGreenDragon** [comment]
HarperZ9, MO2 is for Fallout 4 ONLY at this time. 32 bit files, such as Skyrim, should continue to be run through the 1.3.xx releases. http://www.nexusmods.com/skyrim/mods/1334/?

**Comment by HarperZ9** [comment]
That was my bad LOL I have been up all night I meant F4SE, The only game I am running through the new MO is Fallout 4, I mistyped

**Comment by TheGreenDragon** [comment]
Have you tried MO Alpha 4, released yesterday? It has many stability fixes. https://github.com/TanninOne/modorganizer/releases/tag/v2.0.4a

**Comment by HarperZ9** [comment]
Yeah I discovered the new MO right after the release of 2.0.4 and jumped on it to test some things out. so I may try 2.0.3 after I get some sleep to see if maybe it irons my issue out. Had no idea you guys had something new in the works for 64-bit exe's, so I was excited to happen upon my favorite Mod Management program coming out with a new release.

**Comment by HarperZ9** [comment]
I'll get back to it tomorrow, and drop some more info if I find anything of use

**Comment by Azurelise** [comment]
I've been having a similar issue to Harper as well, I'm using MO 2.0.4 and after I run F4SE the vfs bar loads and a black box for F4SE opens up and that's it, it just stays on the black box.

**Comment by TYGERShrk** [comment]
Did anybody ever figure this out?
I've got it now. Even deleted all my mods and did a re-installation of MO

**Comment by SaviorofTheNorth** [comment]
I know this is an old thread. But for those who are still plagued by this problem, I found a fix for Skyrim SE, delete the usvfs files in your Mod Organizer directory, after you have done so, install the new version of MO 
http://www.nexusmods.com/skyrimspecialedition/mods/6194/?tab=2&navtag=http%3A%2F%2Fwww.nexusmods.com%2Fskyrimspecialedition%2Fajax%2Fmodfiles%2F%3Fid%3D6194&pUp=1 In your original MO directory, it will repair certain files and make them functional again.
